docs/checkit.js

```diff
@@ -162,36 +162,42 @@ CalendarAnalyzer.prototype.getCheckedDaysStreak = function () {
     // Returns the longest streak of checked days
     "use strict";
     
-    var currentStreak = 0,
+    var currentStreak = 1,
         checkedDaysArray = [],
-        lastDate,
         bestStreak = 0,
         i,
         currentDay,
         previousDay,
         dayDiff;
     
-    // place checkedDays object's keys into array and sort the array.
+    // Place checkedDays object's keys into array and sort the array.
     checkedDaysArray = Object.keys(this.calState.checkedDays);
     checkedDaysArray.sort();
-
-    lastDate = checkedDaysArray[0];
+    
+    previousDay = moment(checkedDaysArray[0], "YYYYMMDD");
     
     for (i = 1; i < checkedDaysArray.length; i++) {
```

Comment:
When there's only one element we don't enter this loop, right? How does `bestStreak` end up as `1`? When does it get incremented? I see you have a test for it, but I don't understand why it passes.